<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="pt_MZ">
<context>
    <name>JudgePanel</name>
    <message>
        <location filename="pwv_main.py" line="117"/>
        <source>Contacting the server</source>
        <translation>Contactando o servidor</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="137"/>
        <source>There are no more comparisons to do in this round.</source>
        <translation>Não tem mais comparacões nessa ronda.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="142"/>
        <source>The server returned an action that this application was not able to complete. Action: {}; detail: {}</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="171"/>
        <source>{} seconds have passed. Please select a video now.</source>
        <translation>Já passaram {} segundo. Por favor, faça uma escolha agora.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="182"/>
        <source>Video {}</source>
        <translation>Video {}</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="184"/>
        <source>Play the videos and select an option</source>
        <translation>Assiste ao video e escolha uma opção</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="226"/>
        <source>There was an error starting the video. Please try again.</source>
        <translation>Houve um error ao iniciar do video. Por favor, tente novamente.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="242"/>
        <source>Only {} seconds have passed. You cannot submit your choice until {} seconds have passed.</source>
        <translation>Só passaram {} segundos. Não é possivel fazer uma escolha até {} segundos depois.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="250"/>
        <source>You have not selected an option</source>
        <translation>Você não escolhou uma opção</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="255"/>
        <source>Submitting choice to server</source>
        <translation>Submetendo a sua escolha ao servidor</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="259"/>
        <source>Submitted choice to server</source>
        <translation>A escolha foi submetido ao servidor</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="265"/>
        <source>It was not possible to save the comparison data. Please                      select a new location for the output folder.</source>
        <translation>Não foi possivel salvar os dados da comparação. Por favor, selecione uma nova localização para a pasta de resultados.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="196"/>
        <source>The selected round is no longer available. Please choose a different one</source>
        <translation type="unfinished"></translation>
    </message>
</context>

<context>
    <name>SettingsPanel</name>
    <message>
        <location filename="pwv_main.py" line="314"/>
        <source>There was no response. Your computer may be offline.</source>
        <translation>Não foi possivel conectar ao servidor. Por favor, verfique a sua conexão e tente novamente.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="323"/>
        <source>You are connected to the server. Please sign in again.</source>
        <translation>Está contecado ao servidor. Por favor, inscreva-se novamente.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="328"/>
        <source>You are connected to the server. Please sign in.</source>
        <translation>Está contecado ao servidor. Por favor, inscreva-se.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="335"/>
        <source>It was not possible to connect to the server. Please check your connection and try again.</source>
        <translation>Não foi possivel conectar ao servidor. Por favor, verfique a sua conexão e tente novamente.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="339"/>
        <source>There seems to be an error with the server.</source>
        <translation>Parece que tem um problema com o servidor.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="342"/>
        <source>There was a problem connecting to the server.</source>
        <translation>Houve um problema conectando ao servidor.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="352"/>
        <source>Checking connection...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="366"/>
        <source>You are connected to the server.</source>
        <translation>Está contecado ao servidor.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="368"/>
        <source>You are connected to the server, but do not currently have access to a comparison round.</source>
        <translation>Está conectado ao servidor mas no momento não tem acesso a uma ronda de comparação.</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="380"/>
        <source>Connected to server</source>
        <translation>Conectado ao servidor</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="386"/>
        <source>Check connection / Refresh</source>
        <translation>Verfique a sua conexão / Atualize</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="385"/>
        <source>Not connected to server</source>
        <translation>Não está contecado ao servidor</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="391"/>
        <source>Check server connection</source>
        <translation>Verfique a conexão ao servidor</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="395"/>
        <source>Sign out</source>
        <translation>Sair</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="401"/>
        <source>Signed in as {}</source>
        <translation>Assindao como {}</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="403"/>
        <source>Sign in</source>
        <translation>Assinar</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="413"/>
        <source>You are not signed in</source>
        <translation>Não está assinado</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="458"/>
        <source>Select a folder to save output</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="464"/>
        <source>Select the folder where videos are stored</source>
        <translation>Selecione a pasta na qual estão os videos</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="497"/>
        <source>The comparisons application cannot start because you do not             have access to any rounds.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="505"/>
        <source>Please ensure all settings are configured</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="515"/>
        <source>You need to configure settings before starting the pairwise comparisons application.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="516"/>
        <source>Settings needed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="517"/>
        <source>Return to settings</source>
        <translation>Voltar as configurações</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="518"/>
        <source>Exit application</source>
        <translation>Sair da aplicação</translation>
    </message>
    <message>
        <location filename="pwv_main.py" line="427"/>
        <source>Signing in...</source>
        <translation>Assinando como…</translation>
    </message>
</context>
</TS>
